src/ZeroEventHubClient/Client.cs

```diff
@@ -62,6 +62,26 @@ public Client(string url, int partitionCount, Func<HttpRequestMessage, Task> req
         await FetchEvents(cursors, pageSizeHint, eventReceiver, new List<string>(), cancellationToken);
     }
 
+    /// <summary>
+    /// Fetch events from the server
+    /// </summary>
+    /// <param name="cursor">A single cursor to be used in the request.</param>
+    /// <param name="pageSizeHint">An hint for the page size of the response.
```

Comment:
Mind fixing this typo and in the method above? 
```suggestion
    /// <param name="pageSizeHint">A hint for the page size of the response.
```